src/lightning/pytorch/loggers/logger.py

```diff
@@ -94,7 +94,7 @@ def method(*args: Any, **kwargs: Any) -> None:
 def merge_dicts(  # pragma: no cover
     dicts: Sequence[Mapping],
     agg_key_funcs: Optional[Mapping] = None,
-    default_func: Callable[[Sequence[float]], float] = np.mean,
```

Comment:
`Callable[[Sequence[float]], float]` seems to be Python built-in `float` and not numpy

Reply:
Can you clarify this further @aniketmaurya ?